src/panel/service_panel.js

```diff
@@ -1,47 +1,81 @@
-import ServicePanelView from '../views/service_panel.dot';
-import Panel from '../libs/panel';
-import nconf from '@qwant/nconf-getter';
+/* globals _ */
+import React from 'react';
```

Comment:
Files that use the [JSX syntax](https://reactjs.org/docs/introducing-jsx.html) always need to import the `react` module, event it isn't explicitely called.